web/package.json

```diff
@@ -33,6 +33,7 @@
     "@fortawesome/react-fontawesome": "^0.1.4",
     "@integreat-app/integreat-api-client": "^6.0.0",
     "@integreat-app/react-sticky-headroom": "^1.0.4",
+    "build-configs": "file:../build-configs",
```

Comment:
Command is `yarn workspace web add build-configs@0.0.1`

Reply:
See other answer